crates/astria-sequencer/src/app/tests_breaking_changes.rs

```diff
@@ -138,6 +138,7 @@ async fn app_finalize_block_snapshot() {
     app.finalize_block(finalize_block.clone(), storage.clone())
         .await
         .unwrap();
+    app.commit(storage.clone()).await;
```

Comment:
the app hash is not updated until commit, we were just comparing the genesis app hashes lol.